### DISCUSSION TITLE
Paper-like Dark Mode

### DISCUSSION THREAD

**Discussion by aaronjz** [Akifyss/obsidian-border#273] [Q&A]
I'd love to see a paper-like dark mode. I use the paper-like for light mode, but would love to see the same preset for dark mode. Would someone know how to make one? Maybe one with the colors of "brighter" dark mode. Any help would be appreciated!

**Comment by Akifyss** [comment]
The key is to find a dark mode paper background and enter its URL into the following block:
![image](https://github.com/Akifyss/obsidian-border/assets/23567154/254627da-5a91-435e-bf90-649b3426d08f)
